#1684: Release Scripts (#1685)
AbsaOSS/enceladus: scripts/release/README.md

# Enceladus Release Scripts

## release_prepare

`release_prepare.sh` / `release_prepare.cmd`

> * Merges the current _develop_ branch into the _master_ branch, while removing the `_SNAPSHOT` suffix from the modules' version.

Reviewer: ```suggestion
* Merges the current _develop_ branch into the _master_ branch, while removing the `-SNAPSHOT` suffix from the modules' version.
```

> * Merges the current _develop_ branch into the _master_ branch, while removing the `_SNAPSHOT` suffix from the modules' version.

Reviewer 2: ```suggestion
* Removes the `-SNAPSHOT` version suffix in all submodules in the current _develop_ branch and merges it into the _master_ branch.
```

Because this better communicates the order in which the actions are happening.

* In the _develop_ branch it increases the minor version of all the modules of the project.

> * It commits and pushes the changes.

Reviewer 2: I believe that the final tag version is also created and pushed in the process, which is significant and should be mentioned here, too, for completeness.
